app/CodeFormatter/PhpCodeFormatter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DevCommunityDE\CodeFormatter\CodeFormatter;
+
+/**
+ * Class PhpCodeFormatter.
+ */
+class PhpCodeFormatter extends CodeFormatter
+{
+    /**
+     * @var array
+     */
+    protected static $supported_languages = [
```

Comment:
i recently changed this static variable to a constant

```suggestion
    protected const LANGUAGES = [
```